src/views/shared/container/container.styles.ts

```diff
@@ -0,0 +1,24 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+
+export interface ContainerStyles {
+  background?: boolean;
+}
+
+const useContainerStyles = createUseStyles((theme: Theme) => ({
+  root: {
+    width: "100%",
+    justifyContent: "center",
+    display: "flex",
+    backgroundColor: ({ background }: ContainerStyles) =>
+      background ? theme.palette.white : theme.palette.primary.main,
+  },
+  wrapper: {
+    width: "100%",
+    maxWidth: 700 + 2 * theme.spacing(3.5),
```

Comment:
Why this computation?

Reply:
I brought this from `wallet-ui` because I didn't find any reference to the wrapper

Reply:
> I didn't find any reference to the wrapper

I don't know what you mean by this.

The point is, if we're doing this computation, there must be a reason. I wonder why can't we just set `max-width: 756px;`.

Reply:
Ahh to be equal than the padding

Reply:
This was a tricky thing from the wallet... I can't remember why but it has a reason. But we can remove it from here and set it as Alex said.